src/common/utxobased/network/Socket.ts

```diff
@@ -54,27 +55,29 @@ interface SocketConfig {
   onQueueSpaceCB: OnQueueSpaceCB
 }
 
-interface WsMessage {
-  task: WsTask<unknown>
+interface WsMessage<T> {
+  task: WsTask<T>
   startTime: number
 }
 
-interface Subscriptions {
-  [key: string]: WsSubscription
+interface Subscriptions<T> {
+  [key: string]: WsSubscription<T>
 }
 
-interface PendingMessages {
-  [key: string]: WsMessage
+interface PendingMessages<T> {
+  [key: string]: WsMessage<T>
 }
 
 export function makeSocket(uri: string, config: SocketConfig): Socket {
   let socket: InnerSocket | null
   const { emitter, log, queueSize = 50, walletId } = config
   log('makeSocket connects to', uri)
   const version = ''
-  const subscriptions: Subscriptions = {}
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const subscriptions: Subscriptions<any> = {}
```

Comment:
Why not used "mixed" or "unknown" instead of any? Would that work?

Reply:
`mixed` is a Flow thing. `unknown` would constrain `subscriptions`, we want subscriptions to be unconstrained so it can include any WsSubscription. `subscriptions` is an implementation detail, so it's _slightly_ acceptable if we don't have strict type safety within the implementation?